roles/postgres/tasks/main.yml

```diff
@@ -54,39 +54,47 @@
     state: started
     enabled: true
 
-- name: Configure Alfresco related Databases
+- name: Configure Alfresco databases
   become: true
   become_user: postgres
   vars:
     ansible_ssh_pipelining: yes
   block:
-    - name: "Create a new database with name {{ item }}"
+    - name: "Create necessary databases"
       postgresql_db:
         name: "{{ item }}"
       loop:
         - "{{ repo_db_name }}"
         - "{{ sync_db_name }}"
     
-    - name: "Connect to {{ repo_db_name }} database and set user's password with no expire date"
+    - name: "Give {{ repo_db_username }} user to access {{ repo_db_name }} database with password"
       postgresql_user:
         db: "{{ repo_db_name }}"
         name: "{{ repo_db_username }}"
         password: "{{ repo_db_password }}"
         priv: "ALL"
         expires: infinity
         role_attr_flags: SUPERUSER
-      no_log: true
       tags:
         - molecule-idempotence-notest
     
-    - name: "Connect to {{ sync_db_name }} database and set user's password with no expire date"
+    - name: "Give {{ sync_db_username }} user to access {{ sync_db_name }} database with password"
       postgresql_user:
         db: "{{ sync_db_name }}"
         name: "{{ sync_db_username }}"
         password: "{{ sync_db_password }}"
         priv: "ALL"
         expires: infinity
         role_attr_flags: SUPERUSER
-      no_log: true
+      tags:
+        - molecule-idempotence-notest
+
+    - name: "Revoke {{ repo_db_username }} user to access {{ sync_db_name }} database due to OPSEXP-1373"
```

Comment:
user to access => user access
let's not reference internal issues in code as external users don't have access and can't get its meaning